components/Footer.vue

```diff
@@ -1,9 +1,56 @@
 <template>
 	<footer>
-		<ContactInfo />
-		<SocialMedia />
-		<Funders />
-		<p>
+		<div class="footer-content">
+			<div class="nav-home-about">
+				<div class="nav-link-wrapper">
```

Comment:
What is the purpose of `nav-link-wrapper`? Can we get the same styling without the wrapper? Is this needed because of the sub-nav that isn't implemented yet? If we do need it, is there a more descriptive name that could be used?

Reply:
`nav-link-wrapper` is to add space in between the "home" link and "about" link. This space is also needed when the secondary nav items are in place. My another choice is to remove `nav-link-wrapper` and add padding between link directly. But it will enlarge the link itself and focus/hover styling for links will highlight the whole space, which I feel is less ideal.

Reply:
@jobara, what do you think about renaming "nav-link-wrapper" to "nav-link-block"?

Reply:
See @lliskovoi's comments from https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/133#discussion_r401686538. So there will be block level element containers with the `<li>` that can be used instead. Also we can either use a class like you suggested, or modify the styling to just target the second element. for example you could do something like `.nav-home-about > li  ~ li { margin-top: 2rem }`; which I believe will target all the `<li>` elements that are a direct child of `.nav-home-about` and siblings to the first `<li>`.

Reply:
Css selectors work great. Thanks, @jobara.